test/dnvm.Tests/InstallTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace Dnvm.Tests.InstallTests;
+
+public record InstallLTS(ITestOutputHelper Output, string CommandLine = "--channel lts") : InstallSuccess(Output);
+
+public record InstallCurrent(ITestOutputHelper Output, string CommandLine = "--channel current") : InstallSuccess(Output);
```

Comment:
I tend to avoid inheritance unless it's obviously the best tool for the job. I generally feel like it gets too complicated otherwise and obscures the core code.